syntax = "proto3";

service ProcessService {
  rpc StartProcess (StartProcessRequest) returns (StartProcessResponse);
}

message StartProcessRequest {
  string command = 1; // Command
  int32 planId = 2;   // id
}

message StartProcessResponse {
  string message = 1; // Response message
}